src/main/resources/anniv6Resources/localization/eng/Humility/UIstrings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "${ModID}:Humility": {
+    "TEXT": [
+      "Humility",
+      "All enemies are #rbuffed in new ways. NL Upon entry, obtain a random #grare #grelic [REMOVE_SPACE]."
```

Comment:
You probably saw my message in #modding-technical but posting here as a reminder that I just merged in code that will show the author for each zone and you should add an entry here for that.